cdap-distributions/src/Dockerfile

```diff
@@ -21,26 +21,47 @@
 FROM ubuntu:12.04
 MAINTAINER Cask Data <ops@cask.co>
 
-# Copy scripts
-COPY packer/scripts /tmp/scripts
+# grab gosu for easy step-down from root
+ENV GOSU_VERSION 1.7
+RUN set -x && \
+  apt-get update && \
+  apt-get install -y \
+    curl \
+    git && \
+  curl -vL \
```

Comment:
Nah... curl outputs its own output to STDERR, so you can still see its nice verbose output and redirect the fetched output on STDOUT to a file.